src/main/kotlin/blackjack/domain/Player.kt

```diff
@@ -0,0 +1,44 @@
+package blackjack.domain
+
+class Player(
+    val name: String = "",
+    cards: List<Card> = emptyList()
+) {
+
+    private val _cards = cards.toMutableList()
+    val cards: List<Card> get() = _cards.toList()
+
+    val canReceive: Boolean
+        get() = score() < Blackjack().score
+
+    fun addCard(newCards: List<Card>) {
+        _cards.addAll(newCards)
+    }
+
+    fun addCard(newCards: Card) {
+        _cards.add(newCards)
+    }
+
+
+    private fun aceScore(total: Int): Int {
+        return if (total + (Denomination.ACE.extraScore) <= Blackjack().score)
+            Denomination.ACE.extraScore
+        else Denomination.ACE.score
```

Comment:
```suggestion
        return when (total + (Denomination.ACE.extraScore) <= Blackjack().score) {
            true -> Denomination.ACE.extraScore
            false -> Denomination.ACE.score
        }
```
이건 취향에 따라 다를 수 있는 부분입니다. 전 리턴되는 값들의 indent가 동일한 when 절을 자주 사용하게 되더라구요 😄

Reply:
저도 개인취향은 when절로 condition 상태를 보는것을 좋아하지만 https://kotlinlang.org/docs/coding-conventions.html#if-versus-when 에 3개 이상의 condition 이 있을때 when을 사용하는것이 좋다고 한것같아요.

실제.. 팀원들과 컨벤션을 정할때. 제가 졌던 기억이 있네요^^;